src/components/Calories.js

```diff
@@ -0,0 +1,26 @@
+import React        from 'react'
+import CalorieItem  from './CalorieItem'
+
+function Calories(props){
+    return (
+        <div>
+            <h4 className="title is-4">Today's foods</h4>
+            <ul>
+                {
+                    props.todaysFoods.map((food, i) => {
```

Comment:
some indentation issues in this part (check default space set as 2). Also for returning multiline JSX it is better to start in a line below the return and wrap it in parentheses. 

```
return (
     <CalorieItem 
      key={i} 
      food={food} 
      id={i} 
      onDelete={props.onDelete} 
      />
);
```